tests/Makefile

```diff
@@ -14,7 +14,7 @@ bin:
 bin/%: %.cpp bin
 	$(CXX) $(CFLAGS) $(INC) $(LIBS) $< $(SRC_OBJS) -o $@ $(LINK)
```

Comment:
If you're adding CPPFLAGS, shouldn't  the CPP rule used them?

Reply:
that was made in error